desafio-8/desafio-8.html

```diff
@@ -0,0 +1,44 @@
+<html>
+   <head>
+      <title>Formulário</title>
+      <link rel="stylesheet" href="desafio-8.css">
+   </head>
+   <body>
+      <header>
+         <img scr="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pngwing.com%2Fpt%2Ffree-png-sppmm&psig=AOvVaw2uYMi17rHg4DRGkJ8jbNoK&ust=1644627532953000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCIjpg5W59vUCFQAAAAAdAAAAABAJ" />
+         <h1>Desafio 8 - Formulário</h1>
+      </header>
+      <main>
+         <form>
+            <label class="nomesobrenome">Nome Completo</label>
+            <input type="text" id="nomesobrenome" class="input-padrao">
+            <label for="email">Email</label>
+            <input type="text" id="email" class="input-padrao">
+            <label for="telefone">Telefone</label>
+            <input type="text" id="telefone" class="input-padrao">
+            <input type="submit" value="Enviar formulário" class="button">
+         </form>
+      </main>
+      <footer class="footer">
+         <div class="footer-menu">
```

Comment:
You can use the BEM for all the class names, remember this pattern? 


https://desenvolvimentoparaweb.com/css/bem/

Reply:
Good! I didn't use BEM, I'll adjust